From 7ae0c9d084bb1caf18c751cecb7fb98a619dfbcb Mon Sep 17 00:00:00 2001
From: Zac Medico <zmedico@gentoo.org>
Date: Fri, 15 Dec 2017 12:59:06 -0800
Subject: bin/ebuild-helpers/doins: fix PYTHONPATH setting (bug 640934)

This makes the PYTHONPATH setting consistent with other helpers
like dohtml and xattr/install.

Fixes: d9522ba661b5 ("Rewrite doins in python (bug 624526)")
Bug: https://bugs.gentoo.org/640934
---
 bin/ebuild-helpers/doins | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/ebuild-helpers/doins b/bin/ebuild-helpers/doins
index c3a5789..73c9d8b 100755
--- a/bin/ebuild-helpers/doins
+++ b/bin/ebuild-helpers/doins
@@ -82,7 +82,7 @@ DOINS_ARGS+=(
 # If PYTHONPATH is empty (not unset), it means "add current working directory
 # to the import path" if the Python is prior to 3.4, which would cause
 # unexpected import. See also #469338.
-PYTHONPATH="${PORTAGE_PYM_PATH:-/usr/lib/portage/pym}${PYTHONPATH:+:}${PYTHONPATH}" \
+PYTHONPATH=${PORTAGE_PYTHONPATH:-${PORTAGE_PYM_PATH}} \
 	"${PORTAGE_PYTHON:-/usr/bin/python}" \
 	"${PORTAGE_BIN_PATH:-/usr/lib/portage/bin}"/doins.py \
 	"${DOINS_ARGS[@]}" -- "$@" || \
-- 
cgit v1.1
